tests/specification.test.ts

```
@@ -78,79 +78,63 @@ let UserExtractorModule: typeof import("../src/parser/user-extractor-module").Us
-          return relevance;
-        })()
-      );
-    });
+    config: cfg,
+    logger: new Logs("debug"),
+    octokit: new Octokit(),
+    env: process.env,
+  } as unknown as ContextPlugin;
+  const activity = new IssueActivity(ctx, issue);
 
   activity = new IssueActivity(ctx, issue);
```

Biome (2.1.2)

[error] Can't assign activity because it's a constant.

This is where the variable is defined as constant.

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)